database/migrations/000061_user_invites.up.sql

```diff
@@ -0,0 +1,51 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+-- describes the invite status
+CREATE TYPE invite_status AS ENUM ('pending', 'accepted', 'declined', 'expired', 'revoked');
+-- reflects the roles we have in OpenFGA
+CREATE TYPE user_role AS ENUM ('admin', 'editor', 'viewer', 'policy_writer', 'permissions_manager');
+
+CREATE TABLE IF NOT EXISTS user_invites(
+    id           UUID NOT NULL DEFAULT gen_random_uuid() PRIMARY KEY,
+    email        TEXT NOT NULL,
+    status       invite_status NOT NULL DEFAULT 'pending',
```

Comment:
What are other values of `status` would we use?  (Do we need this column?)

Reply:
I was thinking of the use case where you list the invitations and see some that are pending, some expired, but it makes more sense to make that as separate audit log. Thanks!